corehq/util/__init__.py

```diff
@@ -1,5 +1,6 @@
 import re
 
+from .couch import get_document_or_404
```

Comment:
fwiw, it's not clear to me that we're consistent enough with defining our module apis that these types of changes are actually net positive, but they don't bother me enough to fight either.